src/Operation/Aggregate.php

```diff
@@ -320,7 +318,7 @@ public function getCommandDocument(Server $server)
      *
      * @return array
      */
-    private function createCommandDocument()
+    private function createCommandDocument(): array
```

Comment:
`@return` can be removed. Should we do the same above with `getCommandDocument` (currently has a `@return` annotation but not actual return type hint)?

Reply:
Removed this and a number of other superfluous type annotations.

As for `getCommandDocument`, that method is part of the public API so the return type will have to wait until 2.0.